lib/ecmascript.ts

```diff
@@ -1080,8 +1235,20 @@ export function InterpretTemporalDateTimeFields(calendar, fields, options) {
   return { year, month, day, hour, minute, second, millisecond, microsecond, nanosecond };
 }
 
-export function ToTemporalDateTime(item, options = ObjectCreate(null)) {
-  let year, month, day, hour, minute, second, millisecond, microsecond, nanosecond, calendar;
+export function ToTemporalDateTime(
+  item: Parameters<typeof Temporal.PlainDateTime['from']>[0],
+  options = ObjectCreate(null)
```

Comment:
Does this need explicitly typing as `object` or something else? Last I recall, ObjectCreate returned `any`.

Reply:
All the To* functions get explicit parameter types in a later commit I'm working on, so this problem will go away. I'll resolve for now.